aws-production-2/main.tf

```diff
@@ -273,10 +291,9 @@ module "aws_asg_org" {
 module "aws_asg_org_canary" {
   source                     = "../modules/aws_asg_canary"
   cyclist_auth_token         = "${random_id.cyclist_token_org.hex}"
-  cyclist_url                = "${module.aws_asg_org.cyclist_url}"
+  cyclist_url                = "${module.aws_cyclist_com.cyclist_url}"
```

Comment:
Is this correct for `aws_asg_org_canary` to use the `aws_cyclist_com` cyclist?

Reply:
good catch!